### ISSUE TITLE
charm-tools package should depend on python-pip

### ISSUE THREAD

**Issue by cmars** [juju/charm-tools#12]
Steps to reproduce:
Environment: Ubuntu 14.04, minimal virtual machine image. In my case, I use uvt-kvm to provision throwaway development server instances on my laptop.
1. `apt-add-repository ppa:juju/stable`
2. `apt-get update`
3. `apt-get install charm-tools`

Tried to `charm compose` a layer I'd developed and encountered this error:
```
composer: Composing into /home/ubuntu/oo/charms
composer: Processing layer: layer:basic
composer: Processing layer: oostore
composer: Processing interface: pgsql
Traceback (most recent call last):
  File "/usr/bin/charm-compose", line 9, in <module>
    load_entry_point('charm-tools==1.7.1', 'console_scripts', 'charm-compose')()
  File "/usr/lib/python2.7/dist-packages/charmtools/compose/__init__.py", line 468, in main
    composer()
  File "/usr/lib/python2.7/dist-packages/charmtools/compose/__init__.py", line 393, in __call__
    self.generate()
  File "/usr/lib/python2.7/dist-packages/charmtools/compose/__init__.py", line 359, in generate
    self.exec_plan(self.plan, self.layers)
  File "/usr/lib/python2.7/dist-packages/charmtools/compose/__init__.py", line 337, in exec_plan
    tactic()
  File "/usr/lib/python2.7/dist-packages/charmtools/compose/tactics.py", line 410, in __call__
    spec)).throw_on_error()()
  File "/usr/lib/python2.7/dist-packages/charmtools/utils.py", line 181, in __call__
    p = subprocess.Popen(all_args, **kwargs)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1327, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
make: *** [compose] Error 1
```
With some debug logging, I determined that `python-pip` wasn't installed. `apt-get install python-pip` resolved the issue. This package needs to be added to the charm-tools package dependencies.

**Comment by cmars** [comment]
Tell me where the packaging is, and I'll submit a pull request.. I'm familiar with Debian packaging.

**Comment by marcoceppi** [comment]
Packaging branches aren't hosted anywhere - yet - I've just uploaded a new
build.

**Comment by marcoceppi** [comment]
Fixed in 1.7.1-0ubuntu4

**Comment by cmars** [comment]
Thanks!
